manticore/native/cpu/abstractcpu.py

```diff
@@ -723,7 +743,7 @@ def _raw_read(self, where: int, size: int = 1, force: bool = False) -> bytes:
         assert len(data) == size, "Raw read resulted in wrong data read which should never happen"
         return data
 
-    def read_int(self, where, size=None, force=False):
+    def read_int(self, where, size=None, force=False, publish=True):
```

Comment:
There are times where I did not want events/other hooks to fire when using multiple plugins, so I made it a parameter

Reply:
See this PR https://github.com/trailofbits/manticore/pull/2488